cmd/cluster/aws/create.go

```diff
@@ -11,8 +11,17 @@ import (
 	"github.com/openshift/hypershift/cmd/cluster/core"
 	awsinfra "github.com/openshift/hypershift/cmd/infra/aws"
 	"github.com/openshift/hypershift/cmd/util"
+	hostedclusterinfrastructure "github.com/openshift/hypershift/hypershift-operator/controllers/hci"
 	"github.com/openshift/hypershift/support/infraid"
 	"github.com/spf13/cobra"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	hostedClusterLabel = "openshift.io/hosted-cluster"
```

Comment:
why is this label needed?

Reply:
It makes sure we can easily look up the HCI from the HC name, as technically the hc.Name = hci.Spec...subDomain and hc.Spec.InfraId = hci.Name (though this could be relaxed at some point if we wanted.

Reply:
no longer applied for just CRD